osh/client/completion/_osh-cli

```diff
@@ -139,7 +139,7 @@ _osh-cli_args()
         diff-build|mock-build)
             args+=(
                 '(-d --download-results)'{-d+,--download-results=}'[directory for storing results (leave blank for working directory)]:filename:_files -/'
-                '--install=[install packages into chroot wehan scanning tarballs]:filename:_files'
+                '--install=[install packages into chroot whan scanning tarballs]:filename:_files'
```

Comment:
Do you intentionally spell it `whan` instead of `when`?

Reply:
Not at all.  Thank you for spotting that!

Reply:
Addressed in 1251014e80be949285ff7ba66ff9cd96418620ae.